### PR TITLE
Refactored all services code for converting id into UUID

### PR DESCRIPTION
**Pull request by pallavicoder**
## What  ## 

- Converting ids into UUID from backend 

## Why ##

- To enhance data security and privacy by making it harder for malicious actors to guess or infer object identifiers.

## How ##

- Updated schema.prisma for changing auto-incremented id into UUID
- Updated ledger service APIs
- Updated ecosystem service APIs
- Updated organization service APIs
- Updated user service APIs
- Updated connection service APIs
- Updated agent service APIs
- Updated agent-provisioning service APIs
- Updated issuance service APIs
- Updated verification service APIs
- Updated start_agent shellscript

### CODE REVIEW

File: apps/agent-provisioning/AFJ/scripts/start_agent.sh [hunk 1]
```diff
@@ -15,23 +15,75 @@ PROTOCOL=${12}
...
-  docker network create --driver bridge --subnet 10.20.0.0/16 --gateway 10.20.0.1 credebl-network
-  [ $? != 0 ] && error "Failed to create docker network !" && exit 102
-  echo "CREATED DOCKER NETWORK. SETTING UP INITIAL IPs"
-  INTERNAL_IP=10.20.0.2
-  POSTGRES_IP=10.20.0.3
-  CONTROLLER_IP=10.20.0.4
+echo "-----$AGENCY-----"
+echo "-----$EXTERNAL_IP-----"
+echo "-----$WALLET_NAME-----"
+echo "-----$WALLET_PASSWORD-----"
```
Comment (ajile-in) [inline apps/agent-provisioning/AFJ/scripts/start_agent.sh:22]: DO NOT LOG SECRETS.

File: apps/agent-provisioning/AFJ/scripts/start_agent.sh [hunk 2]
```diff
@@ -15,23 +15,75 @@ PROTOCOL=${12}
...
+echo "-----$RANDOM_SEED-----"
+echo "-----$WEBHOOK_HOST-----"
+echo "-----$WALLET_STORAGE_HOST-----"
+echo "-----$WALLET_STORAGE_PORT-----"
+echo "-----$WALLET_STORAGE_USER-----"
+echo "-----$WALLET_STORAGE_PASSWORD-----"
```
Comment (ajile-in) [inline apps/agent-provisioning/AFJ/scripts/start_agent.sh:28]: DO NOT LOG SECRETS.

**Comment by nishad-ayanworks** [comment]
> SonarCloud Quality Gate failed.    [Quality Gate failed](https://sonarcloud.io/dashboard?id=credebl_platform&pullRequest=232)
> 
> [Bug](https://sonarcloud.io/project/issues?id=credebl_platform&pullRequest=232&resolved=false&types=BUG) [B](https://sonarcloud.io/project/issues?id=credebl_platform&pullRequest=232&resolved=false&types=BUG) [1 Bug](https://sonarcloud.io/project/issues?id=credebl_platform&pullRequest=232&resolved=false&types=BUG) [Vulnerability](https://sonarcloud.io/project/issues?id=credebl_platform&pullRequest=232&resolved=false&types=VULNERABILITY) [A](https://sonarcloud.io/project/issues?id=credebl_platform&pullRequest=232&resolved=false&types=VULNERABILITY) [0 Vulnerabilities](https://sonarcloud.io/project/issues?id=credebl_platform&pullRequest=232&resolved=false&types=VULNERABILITY) [Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=credebl_platform&pullRequest=232&resolved=false&types=SECURITY_HOTSPOT) [A](https://sonarcloud.io/project/security_hotspots?id=credebl_platform&pullRequest=232&resolved=false&types=SECURITY_HOTSPOT) [0 Security Hotspots](https://sonarcloud.io/project/security_hotspots?id=credebl_platform&pullRequest=232&resolved=false&types=SECURITY_HOTSPOT) [Code Smell](https://sonarcloud.io/project/issues?id=credebl_platform&pullRequest=232&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=credebl_platform&pullRequest=232&resolved=false&types=CODE_SMELL) [58 Code Smells](https://sonarcloud.io/project/issues?id=credebl_platform&pullRequest=232&resolved=false&types=CODE_SMELL)
> 
> [No Coverage information](https://sonarcloud.io/component_measures?id=credebl_platform&pullRequest=232) No Coverage information [](https://sonarcloud.io/component_measures?id=credebl_platform&pullRequest=232&metric=new_duplicated_lines_density&view=list) [19.0% Duplication](https://sonarcloud.io/component_measures?id=credebl_platform&pullRequest=232&metric=new_duplicated_lines_density&view=list)
> 
> idea Catch issues before they fail your Quality Gate with our IDE extension sonarlint [SonarLint](https://www.sonarsource.com/products/sonarlint/features/connected-mode/?referrer=sonarcloud-welcome)

Could you please check a bug and try to solve it.

**Review by nishad-ayanworks — COMMENTED** [review]
Please check sonarcloud comment.

**Pull request merged**